crypto/asn1/asn_mime.c

```diff
@@ -833,6 +833,9 @@ static int mime_hdr_addparam(MIME_HEADER *mhdr, const char *name, const char *va
 {
     char *tmpname = NULL, *tmpval = NULL, *p;
     MIME_PARAM *mparam = NULL;
+    if (mhdr == NULL) {
```

Comment:
This looks like a false positive to me.. Coverity doesnt always figure out things correctly when states are used.. In this case I cant see how this can be NULL.

Reply:
Agreed.